packages/amesos2/src/Amesos2_ShyLUBasker.cpp

```diff
@@ -47,331 +47,22 @@
 #include "Amesos2_ShyLUBasker_decl.hpp"
 #include "Amesos2_ShyLUBasker_def.hpp"
 #include "Amesos2_ExplicitInstantiationHelpers.hpp"
+#include "TpetraCore_ETIHelperMacros.h"
```

Comment:
ShyLUBasker should not depend on Tpetra, just Kokkos right ?

Reply:
Correct, ShyLUBasker does not depend on Tpetra, but Amesos2 depends on Tpetra so we are fine with the include here. The ETIHelperMacros clean up the code and help better guard against ETI issues with non-default types enabled.

Reply:
https://github.com/trilinos/Trilinos/blob/cfccfc9ffc274248d8284dc74607ee3e795f16c0/packages/amesos2/cmake/Dependencies.cmake#L5

Reply:
Thanks, @ndellingwood ! I just didn't notice the filename.